### ISSUE TITLE
Mask RCNN Pre-trained Model

### ISSUE THREAD

**Issue by The-Gupta** [CharlesShang/FastMaskRCNN#107]
Can someone share pre-trained model of Mask-RCNN?

**Comment by ishit** [comment]
#89 
#86

**Comment by QtSignalProcessing** [comment]
Hi, I spent 2 months to finish the training of mask-RCNN.

You can download the pre-trained model from the following link.
https://drive.google.com/open?id=0B0J4gcV0gfL4U1NadkllSndKbFk

BTW, it would be really great if anyone could provide the code for testing.

**Comment by The-Gupta** [comment]
Hi @QtSignalProcessing, great that you did and shared. Could you name the dataset you used, and if it's customized, please describe it and tell the number of classes it has? And, could you describe your observations about your model - like how it performs?

**Comment by QtSignalProcessing** [comment]
Hi @The-Gupta, the trained model is named as coco_resnet50....... 
I just followed the readme file to get the result.

I checked the training process for several times and I think it was OK since both the regular loss and total-loss are reducing.

**Comment by The-Gupta** [comment]
And did you test you model's performance manually on images? How well did it perform? Because the loss reduces even in case of overfitting.

**Comment by Zumbalamambo** [comment]
@QtSignalProcessing Can you tell me how do I test this with sample image?

**Comment by anindyaDP** [comment]
@QtSignalProcessing, do you still have the loss weights and the config file for the training setup (libs/config/config_v1.py); I have been training for 300K steps, and the detector seems to be latching more closely to people and ignore other classes; also did you need to customize the train.py script in any way? Thanks
